leftwm/src/bin/leftwm-log.rs

```diff
@@ -0,0 +1,127 @@
+use clap::{arg, command, ArgGroup};
+#[cfg(feature = "file-log")]
+use leftwm::utils::log::file::get_log_path;
+use std::process::{exit, Command};
+
+fn main() {
+    let matches = get_command().get_matches();
+    let follow = matches.get_flag("follow");
+
+    if matches.get_flag("journald") {
+        if cfg!(feature = "journald-log") || matches.get_flag("ignore-build-opts") {
+            journald_log(follow);
+        } else {
+            eprintln!("Failed to execute: leftwm was not built with journald logging");
+            exit(1)
+        }
+    } else if matches.get_flag("syslog") {
+        if cfg!(feature = "sys-log") || matches.get_flag("ignore-build-opts") {
+            syslog(follow);
+        } else {
+            eprintln!("Failed to execute: leftwm was not built with syslog logging");
+            exit(1)
+        }
+    } else if matches.get_flag("file") {
+        #[cfg(feature = "file-log")]
+        file_log(follow);
+        #[cfg(not(feature = "file-log"))]
+        {
+            eprintln!("Failed to execute: leftwm was not built with file logging");
+            exit(1);
+        }
+    } else if cfg!(feature = "journald-log") {
+        journald_log(follow);
+    } else if cfg!(feature = "sys-log") {
+        syslog(follow);
+    } else if cfg!(feature = "file-log") {
+        #[cfg(feature = "file-log")]
+        file_log(follow);
+    } else {
+        eprintln!("Failed to execute: logging not enabled");
+        exit(1);
+    }
+}
+
+fn get_command() -> clap::Command {
+    command!("LeftWM Log")
+        .about("retrieves information logged by leftwm-worker")
+        .help_template(leftwm::utils::get_help_template())
+        .args(&[
+            arg!(-J --journald "use journald log (default)"),
+            arg!(-S --syslog "use syslog (default if built with no journald support"),
+            arg!(-F --file "use file (default if built with no syslog support"),
+            arg!(-i --"ignore-build-opts" "attempt logging regardless of build options"),
```

Comment:
Sorry for my ignorance, but what be the usecase for this flag?
I don't mind adding it, I just don't get it, what situation it would be beneficial in.

Reply:
so i found it useful because for whatever reason my journald log was cloning to my syslog, the previous implementation of the systemd log was super slow, and i somehow found it more preferable to program in an entire  flag rather than change the build options? either way, we can probably remove it without very many people being upset

Reply:
removed in latest commit